File: backend/components/metiers/administration_publique/utils/js/helpers/js_helper.test.py
# js_helper.test.py
# Tests unitaires Python pour js_helper (équivalent JS, ultra avancé)
from .js_helper import to_camel_case

def test_to_camel_case_simple():
    assert to_camel_case('hello_world') == 'helloWorld'

def test_to_camel_case_single():
    assert to_camel_case('hello') == 'hello'

File: backend/components/metiers/administration_publique/utils/js/helpers/js_helper.test.js
// js_helper.test.js
// Tests unitaires JS pour js_helper
const { capitalizeFirst } = require('./js_helper');

describe('capitalizeFirst', () => {
  it('capitalise la première lettre', () => {
    expect(capitalizeFirst('hello')).toBe('Hello');
  });
  it('gère le cas vide', () => {
    expect(capitalizeFirst('')).toBe('');
  });
});
